samples/AspNetODataSample.Web/packages.config

```diff
@@ -15,9 +15,9 @@
   <package id="Microsoft.Bcl.AsyncInterfaces" version="1.0.0" targetFramework="net461" />
   <package id="Microsoft.CodeDom.Providers.DotNetCompilerPlatform" version="1.0.0" targetFramework="net461" />
   <package id="Microsoft.Net.Compilers" version="1.0.0" targetFramework="net461" developmentDependency="true" />
-  <package id="Microsoft.OData.Core" version="7.9.0" targetFramework="net461" />
```

Comment:
any reason to update the dependency version

Reply:
There was a bug which is fixed in 7.9.1 with out which async would throw error in some scenarios